content/en/docs/tasks/access-kubernetes-api/configure-aggregation-layer.md

```diff
@@ -198,6 +198,22 @@ Enable the aggregation layer via the following `kube-apiserver` flags. They may
     --proxy-client-cert-file=<path to aggregator proxy cert>
     --proxy-client-key-file=<path to aggregator proxy key>
 
+### CA Reusage and Conflicts
+
+It is important to note that the Kubernetes apiserver has **two** client CA options:
+
+* `--client-ca-file`
+* `--requestheader-client-ca-file`
+
+Each of these functions independently and can conflict with each other, if not used correctly.
+
+* `--client-ca-file`: When a request arrives to the Kubernetes apiserver, if this option is enabled, the Kubernetes apiserver checks the certificate of the request. If it is signed by one of the CA certificates in the file referenced by `--client-ca-file`, then the request is treated as a legitimate request, and the user is the value of the common name `CN=`, while the group is the organization `O=`. See the [documentaton on TLS authentication](/docs/reference/access-authn-authz/authentication/#x509-client-certs).
+* `--requestheader-client-ca-file`: When a request arrives to the Kubernetes apiserver, if this option is enabled, the Kubernetes apiserver checks the certificate of the request. If it is signed by one of the CA certificates in the file reference by `--requestheader-client-ca-file`, then the request is treated as a potentially legitimate request. The Kubernetes apiserver then checks if the common name `CN=` is one of the names in the list provided by `--requestheader-allowed-names`. If the name is allowed, the request is approved; if it is not, the request is not.
+
+If _both_ `--client-ca-file` and `--requestheader-client-ca-file` are provided, then the request first checks the `--requestheader-client-ca-file` CA and then the `--client-ca-file`. Normally, different CAs, either root CAs or intermediate CAs, are used for each of these options; regular client requests match against `--client-ca-file`, while aggregation requests match against `--requestheader-client-ca-file`. However, if both use the _same_ CA, then client requests that normally would pass via `--client-ca-file` will fail, because the CA will match the CA in `--requestheader-client-ca-file`, but the common name `CN=` will **not** match one of the acceptable common names in `--requestheader-allowed-names`. This can cause your kubelets and other control plane components, as well as end-users, to be unable to authenticate to the Kubernetes apiserver.
```

Comment:
I think it's worth calling out in this doc that the requestheader flags set up an authenticating proxy, and publish that config for consumption by aggregated API servers, but most of this detail about the interaction between x509 and requestheader authenticators is not aggregator-specific, and should go in https://kubernetes.io/docs/reference/access-authn-authz/authentication/#authenticating-proxy and https://kubernetes.io/docs/reference/access-authn-authz/authentication/#x509-client-certs

Reply:
I think it does that [here](https://github.com/kubernetes/website/blob/5b0a08a3c92a3e71ff25f99f1e072e97f5fc742c/content/en/docs/tasks/access-kubernetes-api/configure-aggregation-layer.md#kubernetes-apiserver-client-authentication) from the previous PR. Is that sufficient?